packages/docsearch-react/src/DocSearchButton.tsx

```diff
@@ -25,15 +25,14 @@ export const DocSearchButton = React.forwardRef<
 >(({ translations = {}, ...props }, ref) => {
   const { buttonText = 'Search', buttonAriaLabel = 'Search' } = translations;
 
-  const [key, setKey] = useState<
+  const [key] = useState<
```

Comment:
This can be a `useMemo` now that we don't update the `key` value 🙂

Reply:
Sure, updated.